frontend/components/TableContainer/DataTable/_styles.scss

```diff
@@ -14,6 +14,11 @@ $shadow-transition-width: 10px;
       margin-top: $pad-small;
       flex-grow: 1;
       width: 100%;
+      overflow: scroll;
+
+      @media (min-width: $break-sm) {
+        overflow: unset;
+      }
```

Comment:
Default to scrolling the data table < 768px.

This does mean that any actions pop-up components won't overflow outside the table. In those cases, it's better to override at the component-level and unset the scroll behavior, and manually hide table columns. I do that in the user management and teams management tables.

Reply:
Can you turn this comment into a comment in the code?